src/Renderer/ThreeExtended/GlobeControls.js

```diff
@@ -1122,13 +1135,15 @@ function GlobeControls(camera, domElement, engine) {
     };
 
     // update object camera position
-    this.updateCameraTransformation = function updateCameraTransformation(controlState)
+    this.updateCameraTransformation = function updateCameraTransformation(controlState, forceNoUpdateTarget)
     {
         const bkDamping = this.enableDamping;
         this.enableDamping = false;
         state = controlState || CONTROL_STATE.ORBIT;
         update();
-        updateCameraTargetOnGlobe.bind(this)();
+        if (forceNoUpdateTarget === undefined || forceNoUpdateTarget) {
```

Comment:
Please use positive logic : `forceUpdate`, `preventUpdate`, whatever you prefer. Negative naming is difficult to read and error-prone. Indeed, here when forceNoUpdateTarget is true, we *do* update the target on globe, which seems contradictory to me.

Btw, why do you need to bypass the update of the camera in some case ?